examples/meta/generator/targets/cpp.json

```diff
@@ -122,7 +122,7 @@
         "StaticCall": "C$typeName::$method($arguments)",
         "GlobalCall": "$method($arguments)",
         "Identifier": "$identifier",
-        "Enum":"(machine_int_t)$typeName::$value"
```

Comment:
I got rid of the `machine_int_t` casting stuff for enums, as it won't work with R (there are no enums in R, and swig translates them to strings).